packages/netlify-cms-core/src/actions/__tests__/config.spec.js

```diff
@@ -101,5 +101,55 @@ describe('config', () => {
         ]),
       ).toEqual('_');
     });
+
+    it('should default public_folder to media_folder', () => {
```

Comment:
We should split the collection specific public_folder/media_folder testing to a separate describe block for clarity (same description as top level test).

Reply:
Created a bit of nesting, but still better from before. Also aligned all the test titles to the same format